### DISCUSSION TITLE
Can't reach the entrypage of HACS

### DISCUSSION THREAD

**Discussion by CPONijmegen** [ExperienceLovelace/ha-floorplan#376] [Q&A]
Hey, as stated in the title, I can't reac the enty page of HACS. I only got to this screen, and I can't get any further. Am I doing something wrong?

![Screenshot 2023-11-02 132431](https://github.com/ExperienceLovelace/ha-floorplan/assets/100842318/fb1d5097-4235-47b4-a407-7d88110c5ef9)

**Comment by exetico** [comment]
Search and you'll find ;) 

![billede](https://github.com/ExperienceLovelace/ha-floorplan/assets/3549445/aba4ede5-b3fc-4bea-92ea-fb622991aaf1)

**Reply by exetico** [reply]
HACS updated their GUI, and I've not found time to update the tutorial 😄 .

**Comment by Alsobrookm** [comment]
Im in the same boat. Entry page isnt there. Im missing something.

**Reply by OzGav** [reply]
https://community.home-assistant.io/t/hacs-problem-different-interface/474724
